.circleci/config.yml

```diff
@@ -44,7 +44,7 @@ jobs:
             ./sh/iam-assume-role-and-release.sh  --function ec2-powercycle --alias LIVE
 workflows:
   version: 2
-  build-and-deploy-master:
+  build-and-deploy:
```

Comment:
Just curious. Why we change the workflow config?

Reply:
In this particular case, I think it's better to have a single workflow with a mandatory `build` step and an optional second step, which is either `deploy-master` or `deploy-releases`, rather than having 2 separate workflows that trigger simultaneously, which means that one of two will fail:
![image](https://user-images.githubusercontent.com/3611093/94530428-c3719f80-0243-11eb-8cd2-2595d114ee78.png)